404.md

```diff
@@ -11,6 +11,6 @@ track, maybe you found a broken link. Who knows?
 
 Either way, here's a couple of options for you:
 
-* [Open an issue](https://github.com/docker/docker.github.io/issues/new?title=I%20found%20a%20broken%20link&body=%3CDescribe%20how%20you%20got%20there%3E)
+* [Open an issue](https://github.com/docker/for-win/issues/new)
```

Comment:
I think you are trying to address #6686 with this change but I think that issue refers to a problem in the `docker/for-win` repository, and not the documentation.

This change will make the docs 404 page send users to the docker for Windows repository, which I don't agree with.
Can you revert this back to how it was?

Reply:
I removed this change in the new PR below.